Wire-iOS Tests/AppStateCalculatorTests.swift

```diff
@@ -0,0 +1,262 @@
+//
+// Wire
+// Copyright (C) 2018 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import Wire
+
+final class AppStateCalculatorTests: XCTestCase {
+
+    var sut: AppStateCalculator!
+    var delegate: MockAppStateCalculatorDelegate!
+    
+    override func setUp() {
+        super.setUp()
+        sut = AppStateCalculator()
+        delegate = MockAppStateCalculatorDelegate()
+        delegate.wasNotified = false
+        sut.delegate = delegate
+    }
+
+    override func tearDown() {
+        sut = nil
+        delegate = nil
+        super.tearDown()
+    }
+
+    // MARK: - Tests AppState Cases
+    
+    func testThatAppStateChanges_OnDidBlacklistCurrentVersion() {
+        // WHEN
+        sut.applicationDidBecomeActive()
+        sut.sessionManagerDidBlacklistCurrentVersion()
+
+        // THEN
+        XCTAssertEqual(sut.appState, .blacklisted)
+        XCTAssertTrue(delegate.wasNotified)
```

Comment:
```suggestion
        XCTAssert(delegate.wasNotified)
```
tip: save 4 char to type

Reply:
I think `XCTAssertTrue` is better since the intent is more clear. We also use `XCTAssertTrue` almost everywhere in our existing tests.

Reply:
Actually I was thinking the same. Probably better return to `XCTAssertTrue`